packages/hardhat-core/src/builtin-tasks/flatten.ts

```diff
@@ -135,5 +258,27 @@ task(
     types.inputFile
   )
   .setAction(async ({ files }: { files: string[] | undefined }, { run }) => {
-    console.log(await run(TASK_FLATTEN_GET_FLATTENED_SOURCE, { files }));
+    const [flattenedFile, metadata] = await run(
+      TASK_FLATTEN_GET_FLATTENED_SOURCE_AND_METADATA,
+      { files }
+    );
+
+    console.log(flattenedFile);
+
+    // Warn the user when SPDX licenses or pragma abicoder directives are not specified
+    if (metadata?.filesWithoutLicenses.length > 0) {
+      console.warn(
```

Comment:
This should use `chalk.yellow`, otherwise the warnings are hard to see.

Reply:
done